workflows/utilities/wf_read_QC_trim_se.wdl

```diff
@@ -93,7 +93,7 @@ workflow read_QC_trim_se {
     }
   }
   if ("~{workflow_series}" == "theiaprok") {
-    if (call_midas) {
+    if ((call_midas) && defined(midas_db)) {
```

Comment:
isn't there a midas_db defined by default for theiaprok? i don't remember having to specify this

Reply:
You are very correct! Fixed!